archive/2023-08-30-DssSpell/test/addresses_goerli.sol

```diff
@@ -0,0 +1,437 @@
+// SPDX-FileCopyrightText: © 2020 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.8.16;
+
+contract Addresses {
+
+    mapping (bytes32 => address) public addr;
+
+    constructor() {
+        addr["CHANGELOG"]                       = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+        addr["MULTICALL"]                       = 0xb8c864B60e9467398800Df34da39BF4f0c459461;
+        addr["FAUCET"]                          = 0xa473CdDD6E4FAc72481dc36f39A409D86980D187;
+        addr["MCD_DEPLOY"]                      = 0xc09880a0D6d06fa18C8bDC9dF2E203F0d0124fa1;
+        addr["JOIN_FAB"]                        = 0x0aaA1E0f026c194E0F951a7763F9edc796c6eDeE;
+        addr["FLIP_FAB"]                        = 0x333Ec4d92b546d6107Dc931156139A76dFAfD938;
```

Comment:
Should probably remove this as well:
```suggestion
```

Reply:
To remove it from addresses, we need to remove all lines related to this in *.t.base and config